core/base/inc/TNotifyLink.h

```diff
@@ -93,16 +124,19 @@ class TNotifyLinkBase : public TObject {
 template <class Type>
 class TNotifyLink : public TNotifyLinkBase {
 private:
-   Type *fCurrent;
+   Type *fSubscriber;
```

Comment:
```suggestion
   Type *fNotify;
```
is another option "in line" with the corresponding name/feature in TTree/TChain.

Reply:
What TTree/TChain call `fNotify` is typically a TNotifyLink instance (it cannot be enforced to be unfortunately, but it typically is). So I think it would be confusing to also call `fNotify` the payload of the TNotifyLink, we need different names for different things in order to refer to them from the docs.